.github/workflows/validate.yaml

```diff
@@ -0,0 +1,41 @@
+name: Message Validation
+
+on:
+  pull_request:
+    paths:
+      - "spec/**"
+  push:
+    branches:
+      - 'master'
+      - 'starling-v*-release'
+      - 'v*-release'
+    tags:
+      - 'v*'
+      - 'starling-v*'
+    paths:
+      - "spec/**"
+
+jobs:
+  validation:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout Current Spec
+        uses: actions/checkout@v2
+
+      - name: Checkout Previous Spec
+        uses: actions/checkout@v2
+        with:
+          ref: master
+          path: previous
+
+      - name: Prepare Yaml Comparison
+        run: |
+          mkdir previous-spec && \
```

Comment:
I don't think you need to chain these with `&& \` a la docker, each line is already a separate command